provisioning/templates-api/dev/.air-api.toml

```diff
@@ -3,7 +3,7 @@ tmp_dir = "target/.watcher"
 
 [build]
   bin = "./target/templates/api"
-  args_bin = []
+  args_bin = ["--node-id", "local-node"]
```

Comment:
Vyhodil som default hodnotu, ... nech je to radsej vzdy explicitne zadane cez flag/ENV, je to required.